ansible-scylla-node/tasks/Debian_install.yml

```diff
@@ -19,7 +19,7 @@
     - name: "Validate scylla version correctness"
       ansible.builtin.fail:
         msg: "Too many/few choices for a requested version '{{ scylla_version_to_install }}': {{ aptversions.stdout_lines }}. Bailing out!"
-      when: aptversions.stdout_lines | length != 1
+      when: not scylla_is_installed and aptversions.stdout_lines | length != 1
```

Comment:
You need to add `and not upgrade_version` to your condition, because we still want to run this task to be executed if we're performing an upgrade

Reply:
Fixed, please, re-check, @igorribeiroduarte